README.md

```diff
@@ -32,7 +33,74 @@ $ docker run -p 8000:8000 qr-bot-generator
 
 ## Access the website
 
-- In your browser, navigate to ```https://127.0.0.1:8000.preview.app.github.dev/v1/qr?url=<your-url>```
+- In your browser, navigate to ```https://127.0.0.1:8000.preview.app.github.dev/v1/qr?url=<your-url>``` - localhost, ```https://qr-code-generator.test-80-np.pluto.onglueops.rocks/v1/qr?url=<your-url>``` - Kubernetes deployment
```

Comment:
instead of: `qr-code-generator.test-80-np.pluto.onglueops.rocks`

Use: `qr-code-generator.<captain-domain>` this way it's generic to the environment/cloud the person is working on it against.